.github/workflows/test.yml

```diff
@@ -23,7 +23,7 @@ jobs:
   test:
     strategy:
       matrix:
-        python_version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        python_version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
```

Comment:
```suggestion
        python_version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13-dev"]
```

Reply:
Yes - this does not appear to be necessary ...